logdetective/server/templates/gitlab_comment.md.j2

```diff
@@ -0,0 +1,66 @@
+The package {{ package }} failed to build, here is a possible explanation why.
+
+Please know that the explanation was provided by AI and may be incorrect.
+In this case, we are {{ certainty }}% certain of the response :neutral_face:.
```

Comment:
I'm assuming we'll also template/configure the emoji, right? :grin:

Reply:
Honestly, I hadn't even considered that. @FrostyX didn't templatize that, so I didn't change it.

(And frankly, given the 94% or more confidence it's offering for some _very_ poor answers, the :neutral_face: might be the right choice!)

Reply:
Too late, I just pushed another patch to this set that implements it. Thanks for the pointer, @FrostyX